akvo/rsr/tests/usecases/test_remove_project_parent.py

```diff
@@ -36,7 +39,7 @@ def test_remove_parent(self):
         # When
         command.remove_parent(child_project.object)
         # Then
-        self.assertIsNone(child_project.object.parents_all().first())
+        self.assertIsNone(child_project.object.parent())
```

Comment:
Failed when changed to `self.assertIsNone(Project.objects.get(title='Child project').parent())`

Reply:
Fixed in 086c9ca39ed0327d2c6b33a91903fd69b88318bf